docs/partial_source/contributing/4_open_tasks.rst

```diff
@@ -102,6 +102,21 @@ The general workflow for this task is:
 #. write tests for your function by following the `Ivy Frontend Tests`_ guide
 #. verify that the tests for your function are passing
 
+There are a few points to take note when working on your chosen frontend method:
+
+#. you should only implement **one** frontend function.
+#. the frontend function is framework-specific, thus it should be implemented in
+   its respective frontend framework only.
+#. each frontend method should be tested on all backends to ensure that conversions
+   are working correctly.
+#. type hints, docstrings and examples are not required for frontend functions.
+
+In the case where your chosen method exists in all frameworks by default, but
+does not present in Ivy, please open a GitHub issue requesting the method to be added
```

Comment:
they should actually convert their existing ToDo list issue, rather than opening a new one. This will ensure the function stays as reserved in the ToDo list, and will ensure that this issue is not closed due to inactivity.